streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -198,9 +199,10 @@ public synchronized void close() {
             state = STOPPED;
 
             log.info("Stopped Kafka Stream process");
-        } else {
-            throw new IllegalStateException("This process has not started yet.");
         }
+
+        metrics.close();
```

Comment:
Is it safe to call this multiple times? If not, we would not want to do it if `state == STOPPED`.